### ISSUE TITLE
The number of ConvLSTMCELL in ConvLSTM

### ISSUE THREAD

**Issue by clelouch** [DengPingFan/DAVSOD#19]
Thanks for your code and paper!
I notice that in your implementation, each ConvLSTM contains a single cell. 
Will the performance be improved if more cells are introduced?

**Comment by DengPingFan** [comment]
Hi, 

You can try and share with us. We have no such experiments.

Best,

**Comment by clelouch** [comment]
Thanks for your reply. I notice a typo in the [DAVSOD website](http://dpfan.net/DAVSOD/).
The SegV2 contains 13 clips while the website report 14 clips.
![image](https://user-images.githubusercontent.com/33519394/119464867-c04c9600-bd75-11eb-9afe-79b792c76a35.png)

**Comment by DengPingFan** [comment]
Hi, 
Because the sequence of "penguin" in SegV2 has been deleted by later researchers. In the original paper, there are 14 clips. 

Best,

**Comment by clelouch** [comment]
Many thanks for your help!

**Comment by clelouch** [comment]
Sorry for troubling you. I notice that the DAVIS can be split into a training set and a test set. Could you please tell me how to divide it?

**Comment by DengPingFan** [comment]
Hi,

Please refer to the saliency map results on our home page:
http://dpfan.net/davsod/.  According to these result, you can identify the
training and test set.

Best,


clelouch ***@***.***> 于2021年6月3日周四 上午4:38写道：

**Comment by clelouch** [comment]
Thank you!
